qa/tasks/scripts/ses_nfs_dashboard_rest_api_part2.sh

```diff
@@ -0,0 +1,147 @@
+set -ex
+
+
+curl_cmd (){
+
+# $1 = request type (POST or GET or DELETE)
+# $2 = login_token
+# $3 = url
+# $4 = data
+
+if [ ! -z "$4" ]
+then
+    curl -X $1 -s -H "accept: */*" -H "Authorization: Bearer $2" "${dashboard_addr}$3" -H "Content-Type: application/json" -d "$(cat $4)"
+else
+    curl -X $1 -s -H "accept: */*" -H "Authorization: Bearer $2" "${dashboard_addr}$3"
+fi
+}
+
+rgw_bucket(){
+ # $1 path
+ # $2 pseudo
+ # $3 tag
+cat << EOF > /tmp/rgw_export.json
+{
+  "path": "$1",
+  "cluster_id": "$cluster_id",
+  "daemons": $daemons,
+  "pseudo": "$2",
+  "access_type": "RW",
+  "tag": "$3",
+  "squash": "no_root_squash",
+  "security_label": false,
+  "protocols": [ 3, 4 ],
+  "transports": [ "TCP", "UDP" ],
+  "fsal": {
+    "name": "RGW",
+    "rgw_user_id": "$rgw_user_id"
+  },
+  "clients": [],
+  "reload_daemons": "true"
+}
+EOF
+}
+
+cephfs_bucket () {
+ # $1 path
+ # $2 pseudo
+ # $3 tag
```

Comment:
Same as above, give names to these parameters and use them in dochere template below.